### PR TITLE
Features/fundraiser add program conflict

### PR DESCRIPTION
**Pull request by ryanjgani**
This PR if merged, closes https://github.com/NinjaCompfest/ninja-be/issues/6

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -41,6 +42,9 @@ const userRouter = express.Router();
 userRouter.put("/:id/topup", topup);
 app.use("/users", userRouter);
 
+// fundraiser create program
```
Comment (iamdejan) [inline index.js:45]: remove unused comment

File: controllers/checkAuth.js [hunk 2]
```diff
@@ -4,11 +4,15 @@ const httpStatusCode = require("http-status-codes");
...
     const token = authHeader && authHeader.split(" ")[1];
-    if (token == null)
+    if (token === null) {
         return res.sendStatus(httpStatusCode.StatusCodes.UNAUTHORIZED);
+    }
 
     jwt.verify(token, process.env.JWT_KEY, (err, user) => {
-        if (err) return res.sendStatus(httpStatusCode.StatusCodes.FORBIDDEN);
+        if (err) {
+            console.log(err);
```
Comment (ryanjgani) [inline controllers/checkAuth.js:13]: need to remove

File: repositories/repository.js [hunk 3]
```diff
@@ -48,8 +48,7 @@ class Repository {
 
     // Programs
```
Comment (iamdejan) [inline repositories/repository.js:49]: why commented? lol

File: repositories/repository.js [hunk 4]
```diff
@@ -59,26 +58,37 @@ class Repository {
...
+            { $inc: { balance: amount } },
+            { new: true }
+        ).exec();
         if (user === null) {
             return undefined;
         }
         return user;
     }
+
+    async addProgram(request) {
```
Comment (cholazzzb) [inline repositories/repository.js:79]: move the request to manager... biar seragam aja sama yang lain...

**Pull request merged**